tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss

```diff
@@ -298,33 +298,35 @@ nav.wrapper-preview-menu {
     }
   }
 }
-body.view-statictab, body.view-instructordash, body.view-teams, body.view-wiki {
-  .wrapper-course-material {
-    margin: 0 auto;
-    max-width: 1600px;
-    padding: 20px 15px !important;
-    .tabs.course-tabs {
-      margin: 0 0 10px;
-      border-bottom: 1px solid #E5E7EB;
-      li {
-        margin: 0 32px 0 0;
-        a {
-          border-width: 2px;
-          font-size: 14px;
-          font-weight: 500;
-          color: $light-dark;
-          &:hover, &:visited {
-            color: $dark;
-            border-color: transparent;
-          }
-          &.active {
-            color: $dark;
-            border-color: $primary;
-          }
+.wrapper-course-material[aria-label="Course Material"] {
```

Comment:
nit: any reason for moving it out of all body selectors?

Reply:
Yes, The `wrapper-course-material` is available on all pages (instructors, teams, wiki, static-page). so, if there is notes page and textbook page, we have to add `body.notes` for Notes page. For textbook page, there is no such class with body tag (it has `<body classname="indigo-dark-theme lang_en ltc" />` these classes). So, to apply on `textbook` page, I've made it generic and moved out of body classes. Also made it specific to tabs using the `aria-label` attribute.